node/external/timemachine/fee/memoryFootprint/memory_test.go

```diff
@@ -19,7 +19,7 @@ func TestFeeComputer_MemoryFootprint(t *testing.T) {
 	}
 
 	numEpochs := 10000
-	maxFootprintNumBytes := 50_000_000
+	maxFootprintNumBytes := 60_000_000
```

Comment:
do we have a possible leak that we require this change?

Reply:
reverted this change